src/frontend/README.md

```diff
@@ -1,7 +1,121 @@
-# frontend
+# frontend service
 
-Run the following command to restore dependencies to `vendor/` directory:
+The **frontend** service is responsible for rendering the UI for the store's website.
+It serves as the main entry point for the application routing requests to their 
+appropriate backend service.
+The application uses Server Side Rendering (SSR) to generate HTML consumed by 
+the browser.
 
-```sh
-dep ensure --vendor-only
+The following routes are defined by the frontend:
+
+| Path              | Method | Use                               |
```

Comment:
From a priority perspective do we want the user reading OpenTelemetry content first or service information?

Reply:
Should we put this table under the HTTP instrumentation section instead as the set of routes that will be auto instrumented?